graylog2-web-interface/src/components/content-packs/ContentPackInstall.jsx

```diff
@@ -103,45 +114,51 @@ class ContentPackInstall extends React.Component {
   }
 
   render() {
-    const parameterInput = this.props.contentPack.parameters.map((parameter) => {
+    const { contentPack: contentPackData } = this.props;
+    const { comment } = this.state;
+    const parameterInput = contentPackData.parameters.map((parameter) => {
       return this.renderParameter(parameter);
     });
-    const contentPack = ContentPack.fromJSON(this.props.contentPack);
+    const contentPack = ContentPack.fromJSON(contentPackData);
 
     return (
       <div>
-        <Row>
-          <Col smOffset={1} sm={10}>
-            <h2>Install comment</h2>
-            <br />
-            <br />
-            <Input name="comment"
-                   id="comment"
-                   type="text"
-                   maxLength={512}
-                   value={this.state.comment}
-                   onChange={this._getComment}
-                   labelClassName="col-sm-3"
-                   wrapperClassName="col-sm-7"
-                   label="Comment" />
-          </Col>
-        </Row>
-        {parameterInput.length > 0
-      && (
-      <Row>
-        <Col smOffset={1} sm={10}>
-          <h2>Configure Parameter</h2>
-          <br />
-          <br />
-          {parameterInput}
-        </Col>
-      </Row>
-      )}
+        <form onSubmit={this.onInstall}>
+          <Row>
+            <Col smOffset={1} sm={10}>
+              <h2>Install comment</h2>
+              <br />
```

Comment:
If possible, it would be nice if we used `margin/padding-bottom` instead of `<br/>`